### ISSUE TITLE
Matrix object doesn't return rows correctly

### ISSUE THREAD

**Issue by Pragyaan-6988** [ManimCommunity/manim#726]
## Description of bug / unexpected behavior
For a `Matrix` object of shape `m x n`, the `get_rows` method currently returns `n` rows instead of `m`. So when m > n, only n rows are returned and for m < n, `IndexError` is thrown.

## Expected behavior
Return all rows of the `Matrix` object.

## How to reproduce the issue
<details><summary>Code for reproducing the problem</summary>

```py
class MatrixRowFail1(Scene):
    def construct(self):
        # rows > columns
        m = Matrix([[1, 2], [3, 4], [5, 6], [7, 8]])
        m.set_row_colors(BLUE, GREEN, YELLOW, MAROON)  # only 1st 2 rows are colored
        self.add(m)
```
```py
class MatrixRowFail2(Scene):
    def construct(self):
        # rows < columns
        m = Matrix([[1, 2, 3, 4], [5, 6, 7, 8]])
        m.set_row_colors(BLUE, GREEN)  # throws index error
        self.add(m)
```
</details>

## Additional media files
Scene : MatrixRowFail1
![MatrixRowFail1](https://user-images.githubusercontent.com/73361366/99543604-5d023580-29d9-11eb-8b81-ba0176d1ff99.png)

</details>

## Logs
Scene : MatrixRowFail2
<details><summary>Terminal output</summary>

```
Traceback (most recent call last):
  File "e:\manimce\ce_env\lib\site-packages\manim\__main__.py", line 75, in main
    scene.render()
  File "e:\manimce\ce_env\lib\site-packages\manim\scene\scene.py", line 88, in render
    self.construct()
  File "test.py", line 11, in construct
    m.set_row_colors(BLUE, GREEN)
  File "e:\manimce\ce_env\lib\site-packages\manim\mobject\matrix.py", line 163, in set_row_colors
    rows = self.get_rows()
  File "e:\manimce\ce_env\lib\site-packages\manim\mobject\matrix.py", line 147, in get_rows
    *[VGroup(*self.mob_matrix[i, :]) for i in range(self.mob_matrix.shape[1])]
  File "e:\manimce\ce_env\lib\site-packages\manim\mobject\matrix.py", line 147, in <listcomp>
    *[VGroup(*self.mob_matrix[i, :]) for i in range(self.mob_matrix.shape[1])]
IndexError: index 2 is out of bounds for axis 0 with size 2
```
</details>

## Additional comments
Changing the index passed into ``self.mob_matrix.shape`` from 1 to 0 solves the issue.
https://github.com/ManimCommunity/manim/blob/0aa13c47ba138c81adb3d20c33742699cca561ef/manim/mobject/matrix.py#L147

**Comment by leotrs** [comment]
Hello @Pragyaan-6988 and thanks for the report! This does sound like a silly bug (maybe even a typo). Would you be interested in opening a PR for this?

cc. @ManimCommunity/geometry

**Comment by leotrs** [comment]
Fixed in #734. Thanks @Pragyaan-6988 !
